src/kernel/checks.h

```diff
@@ -5,14 +5,23 @@
 #ifndef BITCOIN_KERNEL_CHECKS_H
 #define BITCOIN_KERNEL_CHECKS_H
 
+#include <optional>
+
 namespace kernel {
 
 struct Context;
 
+enum class SanityCheckError {
+    ERROR_INITITALIZING_ECC,
+    ERROR_GLIBCXX_INSANITY,
```

Comment:
Though it's quite literal here, keeping with precedent, let's not reintroduce this language: https://github.com/bitcoin/bitcoin/pull/5727

The errors are already namespaced as part of the enum class anyway, how about just `SanityCheckError::ERROR_GLIBCXX` ?

Reply:
Fixed as of 69500384a2f7f3d63151f23daee873878cc74c0b